css/src/tokens/themes.scss

```diff
@@ -21,6 +21,7 @@ $themes: (
 		'card-background': $palette-white,
 		'border': $palette-grey-30,
 		'border-accent': $palette-yellow-sand,
+		'border-dark': $palette-grey-100,
```

Comment:
We already have border and control border. Do we really want another?

Reply:
Good idea, updated!